chore: create kubefedcluster cr for cluster to register itself as host and member (#22)
codeready-toolchain/toolchain-common: README.adoc

    . takes a secret of the SA (from the `member`)
    . takes API endpoint and cluster name of the `member` cluster from Kube config
    . goes to `host` profile
    . takes cluster name of the `host` cluster from Kube config
    . creates a secret with the SA token taken from the `member`
    . creates `KubeFedCluster` CR representing the added `member`

Note: We have make target available for this i.e. `make add-member-to-host` in member-operator, host-operator repository

*Scenario 2: Add Host to Member*
```bash
./scripts/add-cluster.sh -t host
```
or

```bash
make add-host-to-member
```

**Steps included:**

    . goes to the cluster with `host` profile i.e. host cluster
    . takes a secret of the SA (from the `host`)
    . takes API endpoint and cluster name of the `host` cluster from Kube config
    . goes to `member` profile
    . takes cluster name of the `member` cluster from Kube config
    . creates a secret with the SA token taken from the `host`
    . creates `KubeFedCluster` CR representing the added `host`

Note: We have make target available for this i.e. `make add-host-to-member` member-operator, host-operator repository

> ==== Using minishift without profiles or any single openshift cluster as host and member clusters

Reviewer: ```suggestion
==== Using a single cluster for both operators host and member
```
